### DISCUSSION TITLE
Collecting suggestions | 建议征集

### DISCUSSION THREAD

**Discussion by ADD-SP** [ADD-SP/ngx_waf#46] [Notification]
You can find descriptions of the features being developed at the links below. If you have any suggestions you can write them here.

* Recommended link: https://docs.addesp.com/ngx_waf/roadmap/overview.html
* Alternate link 1: https://add-sp.github.io/ngx_waf-docs/roadmap/overview.html
* Alternate link 2: https://ngx-waf-docs.pages.dev/roadmap/overview.html

***

您可以点击下列的任意一个链接去了解正在开发的功能，期待您的建议。

* 建议链接: https://docs.addesp.com/ngx_waf/zh-cn/roadmap/overview.html
* 备用链接 1: https://add-sp.github.io/ngx_waf-docs/zh-cn/roadmap/overview.html
* 备用链接 2: https://ngx-waf-docs.pages.dev/zh-cn/todo/roadmap.html

**Comment by Bit-Warrior-X** [comment]
Thanks a lot !
